### DISCUSSION TITLE
Namespaces in scene entities

### DISCUSSION THREAD

**Discussion by ccpowers** [foxglove/community#603] [Q&A]
Currently, we are using `visualization_msgs/Marker` messages in Studio. I'd like to move to using the `SceneEntity` schema to allow annotations, but also want the namespaces feature (which allows us to hide/show markers by namespace in studio). Can namespaces be added to scene entities?

**Comment by lucafei** [comment]
yes, i think this is very useful and can also reduce gpu render consumption
